### ISSUE TITLE
Grid: aria-multiselectable property not set on correct element (4.1.2)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#941]
### Expected behavior
Aria-multiselectable property is set for the element which has a role of grid.  
https://www.w3.org/TR/wai-aria-1.1/#aria-multiselectable

### Actual behavior
The property is set on the table which contains the selectable rows but is a presentational role.
